playbook/app/pb_kits/playbook/pb_advanced_table/docs/_advanced_table_default.md

```diff
@@ -1,7 +1,10 @@
-The AdvancedTable kit takes the table data and automatically renders expandable subrows for nested items to any depth needed. In it's simplest form, the kit has two required props: 
+The AdvancedTable kit takes the table data and automatically renders expandable subrows for nested items to any depth needed. In it's simplest form, __the kit has two required props__: 
```

Comment:
... kit accepts tree data and automatically renders expansion controls for nested subrows, to any depth, based on the data it given.